PYME/localization/FitFactories/__init__.py

```diff
@@ -52,3 +61,20 @@
         pass
     
 resFitFactories.sort()
+
+def import_fit_factory(fit_factory):
+    """Helper function to import a fit factory from just the module name,
+    resolves the location and does the import, returning the module
+
+    Parameters
+    ----------
+    fit_factory : str
+        fit factory module name, e.g. LatGaussFitFR
+
+    Returns
+    -------
+    module
+        loaded fit factory module
+    """
+    pkg = package[fit_factory]
```

Comment:
Leaves the possibility for name collisions in plugins - maybe better to:

simple name -> PYME internal (check first)
fully resolved (i.e. `"my_plugin.my_factory"`) -> from plugin